aggregator/src/binaries/janus_cli.rs

```diff
@@ -155,6 +341,95 @@ async fn install_tracing_and_metrics_handlers(
     Ok((trace_guard, metrics_guard))
 }
 
+async fn generate_global_hpke_key<C: Clock>(
+    datastore: &Datastore<C>,
+    dry_run: bool,
+    id: HpkeConfigId,
+    kem: HpkeKemId,
+    kdf: HpkeKdfId,
+    aead: HpkeAeadId,
+    hpke_config_out_file: Option<&Path>,
+) -> Result<()> {
+    let hpke_keypair = Arc::new(generate_hpke_config_and_private_key(id, kem, kdf, aead)?);
+
+    if !dry_run {
+        datastore
+            .run_tx("generate-global-hpke-key", |tx| {
```

Comment:
```suggestion
            .run_tx("generate_global_hpke_key", |tx| {
```
nit: aligns with our other tx names.

Reply:
ah, nice spot; there are a few test tx's with names that use dashes rather than underscores, but I'll leave them be for now.